### PR TITLE
WIP: Implementing node breadcrumbs

### PR DESCRIPTION
**Pull request by bitamar**
#58 
screen shot 2015-02-04 at 16 35 04

In this issue the breadcrumbs will add to the book content type page.

### CODE REVIEW

File: behat/features/person.feature [hunk 1]
```diff
@@ -15,3 +15,14 @@ Feature: Person
     | Mendelssohn, Moses                               | Dessau                |
     | Friedländer, David                               | Königsberg            |
     | Lindau, Baruch ben Jehuda Löb                    | Hannover              |
+
+  @api
+  Scenario Outline: Verify breadcrumbs exist
+    Given I am an anonymous user
+    When  I visit "<title>" node of type "person"
+    Then  I should see the link "<title>"
```
Comment (amitaibu) [inline behat/features/person.feature]: better check this link under the ``.breadcurmbs`` region

Comment (amitaibu) [inline behat/features/person.feature]: @DavidGizra ^^, you can take this from here

Comment (DavidGizra) [inline behat/features/person.feature]: I think the test need to be in book.feature and not in person.feature.
I will add this test to all the relevant pages.

**Comment by DavidGizra** [comment]
Not have the same style like in the original.
not link the style

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**